collectors/python.d.plugin/varnish/varnish.chart.py

```diff
@@ -168,11 +169,16 @@ def storage_charts_template(name):
                 ['{0}.g_bytes'.format(name), 'allocated', 'absolute', 1, 1 << 10]
             ]
         },
+        order[1]: {
+            'options': [None, 'Storage "{0}" Allocated Objects'.format(name), 'objects', 'storage usage', 'varnish.storage_usage', 'line'],
```

Comment:
I'm not sure about the `varnish.storage_usage` parameter. I used the same as in the previous chart, because both are in the "storage usage" section. Is it correct?

Reply:
It is [chart `context`](https://learn.netdata.cloud/docs/agent/collectors/plugins.d#chart), i think it should be `varnish.storage_alloc_objs`

Reply:
Sorry for the delay @ilyam8 . It is updated now. Does this "chart context" need to be unique? Because if we have this chart for each of the storages, then we will have multiple `varnish.storage_alloc_objs`, right?

Reply:
No, it suposed to be not unique so you can use it in an alarm template and apply alarms for all the chart with the same context.

Reply:
Cool, thanks @ilyam8